plugins.d/alarm-notify.sh

```diff
@@ -1910,6 +1942,12 @@ SENT_FLOCK=$?
 send_discord "${DISCORD_WEBHOOK_URL}" "${to_discord}"
 SENT_DISCORD=$?
 
+# -----------------------------------------------------------------------------
+# send the Alexa notification
+
+send_alexa
+SENT_ALEXA=$?
```

Comment:
This may cause some confusion due to different variable named `SEND_ALEXA`. Please change one of those.